buildenv/jenkins/jobs/pipelines/Pipeline-Build-Test-All.groovy

```diff
@@ -40,10 +40,10 @@
  *              win_x86-64_cmprssptrs,
  *              win_x86 (Java 8 support only),
  *              zos_390-64_cmprssptrs (Java 11 support only),
- *              osx_x86-64 (Java 8 and Java 11 support only),
- *              osx_x86-64_cmprssptrs (Java 8 and Java 11 support only),
- *              aarch64_linux (Java 8 and Java 11 support only),
- *              aarch64_linux_xl (Java 8 and Java 11 support only)
+ *              osx_x86-64,
+ *              osx_x86-64_cmprssptrs,
```

Comment:
The osx platforms are `x86-64_mac` and `x86-64_mac_cmprssptrs`; there's no alias by these names either.